File: bulk/python/xss_01.py
# CWE-79: XSS via Flask with Markup
from flask import Flask, request, Markup
from markupsafe import escape
app = Flask(__name__)
@app.route('/greet')
def greet():
    name = request.args.get('name', '')
    return Markup('<h1>Hello ' + escape(name) + '</h1>')  # unescaped

File: bulk/go/xss_01.go
// CWE-79: XSS via fmt.Fprintf without HTML escaping
package main
import (
	"fmt"
	"net/http"
)
func greet(w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	fmt.Fprintf(w, "<h1>Hello %s</h1>", name) // reflected XSS
}
